src/main/java/uk/gov/hmcts/juror/api/moj/report/DataType.java

```diff
@@ -39,12 +39,21 @@ public enum DataType {
         .then("Not police checked")
         .when(QJuror.juror.policeCheck.eq(PoliceCheck.INELIGIBLE)).then("Failed police check")
         .otherwise(""), QJuror.juror),
+
+    // juror address data types
     POSTCODE("Postcode", String.class, QJuror.juror.postcode, QJuror.juror),
-    POSTPONED_TO("Postcode", LocalDate.class, QJurorPool.jurorPool.deferralDate, QJuror.juror),
+    ADDRESS_LINE_1("Address Line 1", String.class, QJuror.juror.addressLine1, QJuror.juror),
+    ADDRESS_LINE_2("Address Line 2", String.class, QJuror.juror.addressLine2, QJuror.juror),
+    ADDRESS_LINE_3("Address Line 3", String.class, QJuror.juror.addressLine3, QJuror.juror),
+    ADDRESS_LINE_4("Address Line 4", String.class, QJuror.juror.addressLine4, QJuror.juror),
+    ADDRESS_LINE_5("Address Line 5", String.class, QJuror.juror.addressLine5, QJuror.juror),
+    JUROR_ADDRESS("Address", List.class, ADDRESS_LINE_1, ADDRESS_LINE_2, ADDRESS_LINE_3, ADDRESS_LINE_4,
+        ADDRESS_LINE_5, POSTCODE),
```

Comment:
It will be worth naming this ADDRESS_WITH_POSTCODE instead of JUROR_ADDRESS
As sometimes we just want address lines

Reply:
Renamed to juror_postal_address